ee/clickhouse/views/test/test_clickhouse_groups.py

```diff
@@ -0,0 +1,45 @@
+from ee.clickhouse.models.group import create_group
+from ee.clickhouse.util import ClickhouseTestMixin
+from posthog.test.base import APIBaseTest
+
+
+class ClickhouseTestGroupsApi(ClickhouseTestMixin, APIBaseTest):
+    def test_property_definitions(self):
+        create_group(
+            team_id=self.team.pk,
+            group_type_index=0,
+            group_key="org:5",
+            properties={"industry": "finance", "name": "Mr. Krabs"},
+        )
+        create_group(team_id=self.team.pk, group_type_index=0, group_key="org:6", properties={"industry": "technology"})
+        create_group(team_id=self.team.pk, group_type_index=1, group_key="company:1", properties={"name": "Plankton"})
+        create_group(team_id=self.team.pk, group_type_index=1, group_key="company:2", properties={})
+
+        response = self.client.get(f"/api/projects/{self.team.id}/groups/property_definitions").json()
+        self.assertEqual(
+            response,
+            {
+                "0": [{"name": "industry", "count": 2}, {"name": "name", "count": 1}],
+                "1": [{"name": "name", "count": 1}],
+            },
+        )
+
+    def test_property_values(self):
+        create_group(team_id=self.team.pk, group_type_index=0, group_key="org:5", properties={"industry": "finance"})
+        create_group(team_id=self.team.pk, group_type_index=0, group_key="org:6", properties={"industry": "technology"})
+        create_group(team_id=self.team.pk, group_type_index=1, group_key="org:1", properties={"industry": "finance"})
+        response = self.client.get(
+            f"/api/projects/{self.team.id}/groups/property_values/?key=industry&group_type_index=0"
+        ).json()
+        self.assertEqual(len(response), 2)
+        self.assertEqual(response, [{"name": ["finance"]}, {"name": ["technology"]}])
```

Comment:
value of name shouldn't be an array right? Quickly looking at the implementation of other related endpoints that seems like a bug to fix.

Reply:
The other endpoints are not in an array, I thought it might've been something groups-specific so I left it as that, but can fix :)

Reply:
The values come through from the SQL response in a tuple, would we ever have multiple values in that tuple?

Reply:
Taking a look at the sql - it's returning a single column so no :)